use-location.js

```diff
@@ -53,13 +51,10 @@ export default ({ base = "" } = {}) => {
 // is to monkey-patch these methods.
 //
 // See https://stackoverflow.com/a/4585031
+if (typeof history !== "undefined") {
+const eventsToPatch = [eventPushState, eventReplaceState];
 
-let patched = 0;
-
-const patchHistoryEvents = () => {
-  if (patched) return;
-
-  [eventPushState, eventReplaceState].map((type) => {
+for (const type of eventsToPatch) {
```

Comment:
While the `size-limit` does show a difference in size (-4B), I don't believe there'd have been a difference with `terser` or other minifiers.
I have modified the code though to account for the cases where users don't use bundlers and since you prefer it :)